src/common/locales/default.json

```diff
@@ -1809,5 +1809,7 @@
   "instant_mobile_notifications": "Instant mobile notifications:",
   "instant_mobile_notifications_open": "Open the Telegram chart with ",
   "instant_mobile_notifications_hive": " and enter the Hive usernames to subscribe.",
-  "sign_in_error_not_cookies": "You need to disable blocking cookies in incognito mode, in your browser settings"
+  "sign_in_error_not_cookies": "You need to disable blocking cookies in incognito mode, in your browser settings",
+  "power_up_from":"Power up from",
```

Comment:
"power_up_from":"Power up from {from}",
    "power_up_to":"Power up from {to}",